projects/storyteller/webapp/src/app/components/feed/feed.component.ts

```diff
@@ -17,29 +17,23 @@ export class FeedComponent implements OnInit {
     private encodingService: EncodingService,
   ) {
     if (firebaseService.isOnline) {
-      this.loadStory();
-      this.loadscreenshots();
+      zip(
+        this.firebaseService.getStorylistAll(),
+        this.firebaseService.getScreenshotAll()
+      ).subscribe(data => {
+        this.storylist = data[0];
+        this.screenshots = data[1];
+      });
     } else {
       this.firebaseService.anonymousLogin().then(() => {
-        this.loadStory();
-        this.loadscreenshots();
+        zip(
+          this.firebaseService.getStorylistAll(),
+          this.firebaseService.getScreenshotAll()
+        ).subscribe(data => {
+          this.storylist = data[0];
+          this.screenshots = data[1];
+        });
```

Comment:
It's pretty same code, isn't it?
Maybe we can encapsulate it to a method and call it instead?